cross/fontconfig/Makefile

```diff
@@ -0,0 +1,18 @@
+PKG_NAME = fontconfig
+PKG_VERS = 2.11.92
```

Comment:
`.94` is available too: http://www.freedesktop.org/software/fontconfig/release/fontconfig-2.11.94.tar.bz2